t/ui/16-activity-view.t

```diff
@@ -48,20 +49,83 @@ subtest 'Access activity view via the menu' => sub {
     $driver->title_is('openQA: Personal Activity View', 'on activity view');
 };
 
+#                state     result        blocked css class
+# from fixtures
+my %state_result = (
+    99981 => [qw(cancelled skipped            0 fa-times)],
+    99926 => [qw(done      incomplete         0 result_incomplete)],
+    80000 => [qw(done      passed             0 result_passed)],
+    99937 => [qw(done      passed             0 result_passed)],
+    99936 => [qw(done      softfailed         0 result_softfailed)],
+    99938 => [qw(done      failed             0 result_failed)],
+    99927 => [qw(scheduled none               0 state_scheduled)],
+    99928 => [qw(scheduled none               0 state_scheduled)],
+    99963 => [qw(running   none               0 state_running)],
+);
+# created during test
+my %state_result_create = (
+    80001 => [qw(cancelled obsoleted          0 fa-times)],
+    80002 => [qw(cancelled parallel_restarted 0 fa-times)],
+    80003 => [qw(cancelled skipped            0 fa-times)],
+    80004 => [qw(done      obsoleted          0 result_obsoleted)],
+    80005 => [qw(done      parallel_failed    0 result_parallel_failed)],
+    80006 => [qw(done      parallel_restarted 0 result_parallel_restarted)],
+    80007 => [qw(done      skipped            0 result_skipped)],
+    80008 => [qw(done      timeout_exceeded   0 result_timeout_exceeded)],
+    80009 => [qw(done      user_cancelled     0 result_user_cancelled)],
+    80010 => [qw(scheduled none               99963 state_blocked)],
+);
+
 subtest 'Current jobs' => sub {
     my $schema = $t->app->schema;
     my $events = $schema->resultset('AuditEvents');
 
     # The events are interchangeable, but all of these should work
     my %fake_events = (
-        80000 => 'job_done',             # passed
-        99926 => 'job_restart',          # incomplete+reason
-        99927 => 'job_update_result',    # scheduled,
-        99936 => 'job_create',           # softfailed
-        99937 => 'job_done',             # passed,
+        80000 => 'job_done',
+        99926 => 'job_restart',
+        99927 => 'job_update_result',
+        99936 => 'job_create',
+        99937 => 'job_done',
+        99981 => 'job_create',
+        99928 => 'job_create',
+        99963 => 'job_create',
+        99938 => 'job_create',
+        80001 => 'job_create',
+        80002 => 'job_create',
+        80003 => 'job_create',
+        80004 => 'job_create',
+        80005 => 'job_create',
+        80006 => 'job_create',
+        80007 => 'job_create',
+        80008 => 'job_create',
+        80009 => 'job_create',
+        80010 => 'job_create',
```

Comment:
Maybe you can just loop through the numbers for the create events?

Reply:
done